### ISSUE TITLE
Add Thai Language support in Meilisearch

### ISSUE THREAD

**Issue by ManyTheFish** [meilisearch/meilisearch#2686]
Officially support the Thai language in Meilisearch:

- [x] Implement Thai in Charabia
- [x] Integrate Charabia in milli
- [ ] Integrate the new version of milli into Meilisearch

See more on [crates.io/crates/charabia](https://crates.io/crates/charabia)

**Comment by curquiza** [comment]
Pinging @meilisearch/docs-team 🎉

**Comment by curquiza** [comment]
Hello everyone watching this issue!

We have just released v0.29.0rc1, which is a release candidate of v0.29.0 🔥 
It means you can test the just-introduced Thai language support! Binaries are attached to the release, or you can you the docker image:
```bash
docker run -it --rm \
    -p 7700:7700 \
    getmeili/meilisearch:v0.29.0rc0
```

Let us know about any bugs or feedback! 😄 It would be really helpful.

FYI, the official v0.29.0 release will be available on 3rd October.
